.github/workflows/rust.yml

```diff
@@ -2,33 +2,32 @@ name: Build and Test Nova
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
```

Comment:
@nalinbhardwaj thanks for the PR! 

What is the change here? It appears to be mostly a formatting change, except for the clippy line. Could we make the diff reflect the actual change?

Reply:
Oops, my editor auto linted the file, undid the lint. The change here is to remove `--all-features` (since [GitHub Workflows doesn't support GPU CUDA/OpenCL](https://github.com/github/roadmap/issues/505) the tests are expected to fail for those features)